api/dataprocessor.go

```diff
@@ -414,11 +414,61 @@ func (s *Server) getSeries(req models.Req, consolidator consolidation.Consolidat
 		// if to < oldest -> no need to search until oldest, only search until to
 		until := util.Min(oldest, toUnix)
 		logLoad("cassan", key, fromUnix, until)
-		storeIters, err := s.BackendStore.Search(key, fromUnix, until)
-		if err != nil {
-			panic(err)
+
+		log.Debug("cache: searching query key %s, from %d, until %d", key, fromUnix, until)
```

Comment:
didn't notice that there's a standard for these prefixes. also fixed the ones in `mdata`